awswrangler/distributed/ray/_core.py

```diff
@@ -24,27 +24,32 @@ def __init__(
     ):
         logging.basicConfig(level=log_level, format=format, datefmt=datefmt)
 
-    def get_logger(self, name: Union[str, Any] = None) -> Union[logging.Logger, Any]:
+    def get_logger(self, name: Union[str, Any] = None) -> Optional[logging.Logger]:
         """Return logger object."""
-        return logging.getLogger(name) if engine.get() == EngineEnum.RAY else None
+        if config.log_to_driver:
```

Comment:
I am not thrilled with this way of controlling how to turn the logger on and off. This is because we also use this config flag in our `ray_initialize` call (via [@applyconfig](https://github.com/aws/aws-sdk-pandas/blob/release-3.0.0/awswrangler/distributed/ray/_core.py#L72)). It means that:
1. Logging must be configured before library import by setting the WR_LOG_TO_DRIVER variable to True or False
2. Once set, changing `wr.config.log_to_driver` to True is not enough because a new call to `ray_initialize` must be made for the change to take effect

Reply:
Can we put an explicit `bool` parameter in the method to enable `log_to_driver` in addition to the config item?